department-of-veterans-affairs/va.gov-cms
# VACMS-9612: fetch lat/long from facility API for all VHA facilities

## Description

Closes #9612
Closes #9627 

## Testing done

Manual testing. I ran each of the updated migrate scripts locally to be sure the lat/long values were brought in properly. It is worth noting that the coordinates in the Facility API are listed longitude first and latitude second. I learned this after manually looking up the lat/long values for some of the addresses online. 

## Screenshots

NCA Facility Example:

![Screen Shot 2022-07-20 at 4 32 33 PM](https://user-images.githubusercontent.com/21045418/180080443-ec661ca7-a708-46aa-8e5a-ab284071dda3.png)

VAMC Facility Example:

![Screen Shot 2022-07-20 at 4 50 10 PM](https://user-images.githubusercontent.com/21045418/180081004-4c4ae9c7-a57e-41c8-ac4e-e977a828039c.png)

VBA Facility Example:

![Screen Shot 2022-07-20 at 4 35 03 PM](https://user-images.githubusercontent.com/21045418/180080608-9d471e2c-c55b-4b48-a996-98fa3f4eb1db.png)

Vet Center Example:

![Screen Shot 2022-07-20 at 4 45 11 PM](https://user-images.githubusercontent.com/21045418/180080913-dc70f6d8-a874-4bae-afae-0d3c34c0917e.png)

Vet Center Outstation Example:

![Screen Shot 2022-07-20 at 4 38 15 PM](https://user-images.githubusercontent.com/21045418/180080765-adc753a4-c53d-4203-9caf-44ddd06f69f3.png)

## QA steps

Likely needs to be performed on a local environment:

1. Connect to the Facility API sandbox endpoint: https://sandbox-api.va.gov/services/va_facilities/v0/facilities/all (I used the Postman application for this)
2. Run each of the following migration commands in a terminal window:
3. ddev drush migrate:import va_node_health_care_local_facility
4. ddev drush migrate:import va_node_facility_nca
5. ddev drush migrate:import va_node_facility_vba
6. ddev drush migrate:import va_node_facility_vet_centers
7. ddev drush migrate:import va_node_facility_vet_centers_os
8. As an admin user, navigate to admin/content and filter by each of the following types. Pick an item from each set of results and look it up in the Facility API dataset to verify the lat/long values were pulled in properly.
9. VAMC Facility
10. NCA Facility
11. VBA Facility
12. Vet Center
13. Vet Center Outstation

### Select Team for PR review
